app/controllers/user_game_pairs_controller.rb

```diff
@@ -37,6 +37,52 @@ def create
     end
   end
 
+  ## Controller for "Going" button on games index page
+  def rsvp_going
+    game = Game.find(params[:game_id])
+
+    @user_game_pair = UserGamePair.new
+    game.curr_size = game.curr_size + 1
```

Comment:
Do we want to have a check here at this stage to make sure curr_size >= max_size?

Reply:
Ya I'll fix that now